lscolors.go

```diff
@@ -0,0 +1,148 @@
+package main
+
+import (
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/nsf/termbox-go"
+)
+
+func init() {
+	if lscolors := os.Getenv("LSCOLORS"); lscolors != "" {
+		lsColors.parseBSD(lscolors)
+		gOpts.lscolors = true
+	}
+	if ls_colors := os.Getenv("LS_COLORS"); ls_colors != "" {
+		lsColors.parseLinux(ls_colors)
+		gOpts.lscolors = true
+	}
+}
+
+type lsColorsEntry struct {
+	fg termbox.Attribute
+	bg termbox.Attribute
+}
+
+type lsColorsT map[string]lsColorsEntry
+
+var lsColors = make(lsColorsT)
+
+// This function parses LS_COLORS environment variable
+func (lsc lsColorsT) parseLinux(env string) {
```

Comment:
This function only seems to handle the basic 8 colors though `LS_COLOR` can handle 256 colors. There is a similar logic implemented in [ui.go](https://github.com/gokcehan/lf/blob/5a88ca59a34b2eb6ea56a35602cd8c74091c3e72/ui.go#L129) which can also handle 256 colors. Ideally these two should be merged with some refactoring.

Reply:
Will try to figure this out